tests/test_simple.py

```diff
@@ -14,6 +14,9 @@
 
```

Comment:
If both parsers are available, you should be running the same set of tests against each.

Reply:
Done.